packages/components/src/components/badge/badge.stories.ts

```diff
@@ -1,14 +1,24 @@
 import '../../solid-components';
+import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
 import { withActions } from '@storybook/addon-actions/decorator';
 
 const { argTypes, parameters } = storybookDefaults('sd-badge');
 const { overrideArgs } = storybookHelpers('sd-badge');
 const { generateTemplate } = storybookTemplate('sd-badge');
 
+/**
+ *
+ * The badge can be used to display a number or status. It can be used in combination with other components to draw attention to specific information.
```

Comment:
```suggestion
 * The badge is a non-interactive element that can be used to display a number or status. It can be used in combination with other components to draw attention to specific information.
```

Reply:
We decided to go with the description from the Figme documentation